source/Shader.cpp

```diff
@@ -151,3 +306,15 @@ GLuint Shader::Compile(const char *str, GLenum type)
 
 	return object;
 }
+
+
+
+string Shader::ShaderPath(string shader, bool isFragment, bool isInBuilt, bool useShaderSwizzle)
+{
+	string path = Files::Shaders();
+	path += isInBuilt ? "inbuilt/" : "";
+	path += shader;
+	path += useShaderSwizzle ? "/swizzle/" : "/";
```

Comment:
You're appending swizzle to the shader paths, this only works for the sprite shader and for every other the path will be incorrect.